src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -3,266 +3,300 @@
 // The full license information can be found in LICENSE in the root directory of this project.
 
 @include exports('datagrid.clarity') {
-    @include basic-table(".datagrid", ".datagrid-head", ".datagrid-body",
-        ".datagrid-row", ".datagrid-column", ".datagrid-cell");
-
-    $clr-datagrid-icon-size: 0.583333rem;
-    $clr-datagrid-action-arrow-size: 0.25rem;
-    $clr-datagrid-fixed-column-size: (2 * $clr-table-cellpadding) + $clr-table-lineheight;
-    $clr-datagrid-column-handle-tracker-top: -0.5rem;
-    $clr-datagrid-column-separator-expandby: 0.5rem;
-    $clr-datagrid-compact-outline-offset: -0.1666667rem;
+    @include basic-table(".datagrid", ".datagrid-header", ".datagrid-body",
+            ".datagrid-row", ".datagrid-column", ".datagrid-cell");
 
     .datagrid-host {
         display: flex;
         flex-flow: column nowrap;
-    }
-
-    .datagrid-overlay-wrapper {
-        display: flex;
-        flex-direction: row;
-        flex: 0 auto; // Same as flex: initial. It sizes the item based on width/height properties or its content.
+        height: auto;
+        position: relative; // for the .datagrid-spinner to work.
```

Comment:
Damnit, I didn't see this `position: relative`.  That will cut off any filters popover, action overflow menus and column picker... I'm not sure I want to block the PR on this right now, but can you put a `TODO` or `FIXME` on this so we can re-investigate at the "end" of this whole topic branch?